modules/javafx.graphics/src/test/java/test/javafx/scene/image/ImageTest.java

```diff
@@ -154,6 +155,35 @@ public void loadImageAsyncProgressTest() {
         assertTrue(p3 == p4);
     }
 
+    @Test
```

Comment:
Do you think it would make sense to add a test that actually loads a valid image?

Reply:
I don't think that can be done with the stub toolkit  This test is specifically stubbing/mocking everything out to verify that  `Image` works correctly.  However, there are many areas where (real) images are loaded already, and all the code involved (excluding the new parts in `Image`) was already there in one way or another as the URL variants that allowed background loading is also just an `InputStream` with some extra steps to determine their size (for progress).